docs/src/main/asciidoc/dev-services.adoc

```diff
@@ -0,0 +1,75 @@
+////
+This guide is maintained in the main Quarkus repository
+and pull requests should be submitted there:
+https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc
+////
+= Dev Services Overview
+:extension-status: preview
+
+include::./attributes.adoc[]
+
+Quarkus supports the automatic provisioning of unconfigured services in development and test mode. We refer to this capability
+as Dev Services. From a developers perspective this means that if you include an extension and don't configure it then
+Quarkus will automatically start the relevant service (usually using Test Containers behind the scenes) and wire up your
+application to use this service.
+
+All this functionality is part of the Quarkus `deployment` modules, so does not affect the production application in any
+way.
+
+This page lists all the Dev Services that Quarkus supports.
+
+NOTE: In order to use Dev Services you will generally need a working docker environment (remote environments are supported).
+If you don't have docker installed you will need to configure your services normally.
+
+== Amqp
+
+Amqp will be enabled by adding `quarkus-smallrye-reactive-messaging-amqp` to your application.
```

Comment:
I'm not sure about the wording here (and in all the other places). It's not clear it's the Dev Service that is enabled. Also I read that as you add the extension to get the dev service but really you add the extension to get the feature and you have the dev service enabled on the side.

Maybe something like:
> The AMQP Dev Service will be enabled when `quarkus-smallrye-reactive-messaging-amqp` is present in your application.

Note that it might just be me not being a native English speaker.